### PR TITLE
decrypt env vars

### PR DESCRIPTION
**Pull request by MancunianSam**
- Decrypt environment variables with kms
- Return kms response based on the request
- Pass the endpoint to the kms client

### CODE REVIEW

File: src/main/scala/uk/gov/nationalarchives/consignmentexport/authoriser/Lambda.scala [hunk 1]
```diff
@@ -3,10 +3,10 @@ package uk.gov.nationalarchives.consignmentexport.authoriser
 import java.io.{InputStream, OutputStream}
 import java.nio.charset.Charset
 import java.util.UUID
-
 import cats.effect.{Blocker, ContextShift, IO, Resource}
 import cats.implicits._
 import com.nimbusds.oauth2.sdk.token.BearerAccessToken
+import com.typesafe.config.ConfigFactory
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/consignmentexport/authoriser/Lambda.scala:9]: Don't think this import is needed? `import com.typesafe.config.ConfigFactory`

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -1,3 +1,9 @@
 api {
-  url = "http://localhost:9001/graphql"
+  url = "aHR0cDovL2xvY2FsaG9zdDo5MDAxL2dyYXBocWw=" # http://localhost:9001/graphql
```
Comment (TomJKing) [inline src/test/resources/application.conf:2]: Is it worth adding a bit more to the comment to say this is the decoded string?

File: src/test/resources/application.conf [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+}
+kms {
```
Comment (TomJKing) [inline src/test/resources/application.conf:4]: Should all the values be encoded? If not maybe worth mentioning this somewhere

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor comments

File: src/main/scala/uk/gov/nationalarchives/consignmentexport/authoriser/Lambda.scala (hunk 1 above)
Comment (MancunianSam) [reply src/main/scala/uk/gov/nationalarchives/consignmentexport/authoriser/Lambda.scala:9]: I've taken it out.

File: src/test/resources/application.conf (hunk 3 above)
Comment (MancunianSam) [reply src/test/resources/application.conf:4]: I've put the same comment from before in the README.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**